Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -485,8 +485,12 @@ namespace AZ
                 WaitAndCleanCompletionJob(m_simulationCompletion);
             }
 
-            auto taskGraphActiveInterface = AZ::Interface<AZ::TaskGraphActiveInterface>::Get();
-            m_taskGraphActive = taskGraphActiveInterface && taskGraphActiveInterface->IsTaskGraphActive();
+            // Profiling Ros Con demo, Task Graph was 2-3ms slower than Jobs
+            // Time for Jobs was ~5.5ms, maxing out at ~6.3ms
+            // Task Graph took ~7.7ms, maxing out at ~8.7ms
+            // Disabling task graph for performance reasons, to re-enable uncomment these two lines
+            // auto taskGraphActiveInterface = AZ::Interface<AZ::TaskGraphActiveInterface>::Get();
+            // m_taskGraphActive = taskGraphActiveInterface && taskGraphActiveInterface->IsTaskGraphActive();
```

Comment:
Please use a CVar or a Setting in registry rather than commenting out code.

Reply:
+1

Reply:
+1 for the cvar or setting registry. There is already a cvar to switch off TaskGraph as the default.